projects/plugins/jetpack/extensions/blocks/ai-assistant/hooks/use-ai-feature/index.ts

```diff
@@ -25,6 +25,7 @@ export const AI_Assistant_Initial_State = {
 	currentTier: {
 		value: aiAssistantFeature?.[ 'current-tier' ]?.value || 1,
 	},
+	nextTier: aiAssistantFeature?.[ 'next-tier' ] || null,
```

Comment:
I think we should get rid of the `null` part.

```suggestion
	nextTier: aiAssistantFeature?.[ 'next-tier' ],
```